app/views/shared/_article_promos.html.erb

```diff
@@ -8,7 +8,12 @@
       <div class="l-promo-articles__item">
         <div class="t-article-promos">
           <%= link_to item['url'], class: 'promo__link' do %>
-            <%= image_tag(item['image'], alt: '', class: 'promo__img') %>
+            <%= image_tag(item['image'],
+              alt: '',
+              sizes: '(max-width: 479px) 100vw, (max-width: 959px) 50vw, 25vw',
```

Comment:
**I am not advocating anything**. In order to give you the info on the best place to put this line, I need to understand all the implications of it. That's the point of my questions. But I didn't understand one thing: you said that if this line is removed **the srcset will not work** at the viewports and in the other you said there will not be any catastrophic failure. Could not be catastrophic, but is still a failure and the functionality will not work as expected ... right?